RevenueCatUI/Templates/MultiPackageTemplate.swift

```diff
@@ -0,0 +1,241 @@
+import RevenueCat
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+struct MultiPackageTemplate: TemplateViewType {
```

Comment:
The template view itself. Just around ~200 lines, as simple as I could make it using mostly reused components no.
There's still a bit more that can be extracted when we add the second multi-package template.